SECURITY.md

```diff
@@ -0,0 +1,26 @@
+# Security Model
+
+
+*Nakadi* makes use of Oauth2 to provide security, there are three modes that can be 
+configured. In order to configure security for *Nakadi* you need to edit the *Nakadi* configuration file and add oauth2
+ validation endpoints. This is set via two environmental variables.
+ 
+ * AuthenticationMode: Security model to use 
+ 
+ * TokenInfoUrl: oauth2 endpoint to get token information
+
+## OpenAuthenticationMode
```

Comment:
Maybe add the value here (NONE)? Same remark for the next 2 sections, BASIC and FULL